hybrasyl/Objects/VisibleObject.cs

```diff
@@ -46,6 +46,9 @@ public class VisibleObject : WorldObject
         public List<string> DeathPileAllowedLooters { get; set; }
         public DateTime? DeathPileTime { get; set; }
 
+        public List<string> MonsterDeathPileAllowedLooters { get; set; }
+        public DateTime? MonsterDeathPileTime { get; set; }
```

Comment:
I am not sure a differentiation here is needed (e.g. use `DeathPileAllowedLooters`).